FinalTask/src/main/java/file/FileRead.java

```diff
@@ -0,0 +1,39 @@
+package file;
+
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FileRead {
+    public static Object[][] getCredentials() throws IOException {
+        HashMap<String, String> map = new HashMap<String,String>();
```

Comment:
Always use name of interface, when creating objects. It makes code less dependent